Oy2 26709

bflynn-cms opened the pull request:
Story: https://qmacbis.atlassian.net/browse/OY2-26709
Endpoint: See github-actions bot comment

### Details

Change to withdraw rai status text

### Changes

- replaced all instances
- added mapping to ui map
- did map translation on lookup for detail and package view (same as we do for currentStatus)

bflynn-cms committed:
Updated status map to also account for substatus

While I think having the status map include the substatuses is fine, it would also be prudent given a longer time horizon to rebaseline the actual status as well. However not a big deal since that is the purpose of the map, you can have  a different backend status vs what shows to the user.

bflynn-cms committed:
set mapped status after call to actionDelegate

bflynn-cms committed:
update all old statuses and migrate

kristin-at-theta reviewed (approved):
linked story is incorrect, but I figured it out :P 

things look correct, and I like that we changed it fundamentally rather than incrementally.  in VAL, need to migrate, but PROD should be fine with just the rebuild, right? :)

bflynn-cms commented:
> linked story is incorrect, but I figured it out :P
> 
> things look correct, and I like that we changed it fundamentally rather than incrementally. in VAL, need to migrate, but PROD should be fine with just the rebuild, right? :)

Ah yes, I switched the 0 and 7.....updated the link just in case someone else comes along.
And yes i included in this commit the migrate script which should run in val automatically, else ill just trigger it and since PROD doesnt run migrate script automatically we are good there.
